package.json

```diff
@@ -12,7 +12,7 @@
     "build": "cpy src/*.scss lib && rollup -c",
     "watch": "cpy src/*.scss lib && rollup -c -w",
     "release": "np",
-    "docs": "docsify serve docs",
+    "docs": "cd website && npm start",
```

Comment:
Now I'm confused :/
Does docusaurus use `yarn` or `npm`?

Reply:
Docusaurus uses yarn

Reply:
but this is our script

Reply:
Thanks for explaining